src/contracts/Event/DispatchParsingData.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Rest\Event;
+
+final class DispatchParsingData
```

Comment:
naming: can we keep the convention we had so far, something like `BeforeParseEvent`?